packages/web/src/components/pages/Home.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useEffect } from 'react';
+import { Error, Home } from 'components/layouts';
+import { trackCustomEvent } from '../../services/analytics';
+import {
+  useAppDispatch,
+  parseWebsite,
+  resetError,
+  useAppSelector,
+  homeDefaultSelector,
+} from '../../store';
+import { useNavigate } from 'react-router-dom';
+
+export function HomePage() {
+  const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+  const state = useAppSelector(homeDefaultSelector);
+
+  const handleDetectStart = useCallback(async (data: { address: string }) => {
+    trackCustomEvent('HomePage', 'WebsiteSubmitted');
```

Comment:
Why do we track `WebsiteLoaded` inside redux, but `WebsiteSubmitted` inside a component?

Reply:
That's why I left a TODO there :) Usually event tracking is located in separate redux middleware, but I think we better discuss this.